src/createActions.js

```diff
@@ -0,0 +1,59 @@
+import createAction from './createAction';
+
+function formatType(options, key) {
+  return `${options.prefix}${key}`;
+}
+
+/**
+   @param actionMap object optional - creates actions for each key,
+   using the value to determine the payload and meta creators. If
+   value is a fn, use it for the payloadCreator. If value is an
+   object use its `payload` and `meta` keys for payload and meta
+   creators.
+   @param arrayDefTypes array of strings optional
+   @param options object optional
+   @param options.prefix - string prefix all action types with, default ''
+   @returns object of actions
+*/
+export default function createActions(actionMap, arrDefTypes, options) {
+  // adjust for optional arguments
+  if (Array.isArray(actionMap)) { // actionMap not provided, shift
+    options = arrDefTypes;
+    arrDefTypes = actionMap;
+    actionMap = {};
+  }
+
+  if (!Array.isArray(arrDefTypes)) { // arrDefTypes not provided, shift
+    options = arrDefTypes;
+    arrDefTypes = [];
+  }
+
+  if (!options) { options = {}; }
```

Comment:
Since we're using ES6, you should probably use a default parameter `options = {}` here.